playbooks/customize.yml

```diff
@@ -9,7 +9,7 @@
       default: "~/.ssh/id_rsa"
       private: no
     - name: vpn_clients
-      prompt: "How many VPN client profiles should be generated per-service? Press enter for default "
+      prompt: "How many VPN client profiles should be generated per-service (min: 1 max: 20)? Press enter for default "
```

Comment:
Gosh I wish we could use vars in prompt strings :'(

Reply:
If everything was perfect life would be boring ;)